src/Products/PloneMeeting/tests/testMeetingItem.py

```
@@ -7217,18 +7217,18 @@ def test_pm_DisplayOtherMeetingConfigsClonableTo(self):
                     cfg2Title, noneTheoricalMeeting,
                     cfg3Title, noneTheoricalMeeting),
                     'utf-8'))
-        cleanRamCacheFor('Products.PloneMeeting.MeetingConfig.getMeetingsAcceptingItems')
+        cleanRamCacheFor('Products.PloneMeeting.content.meetingconfig.getMeetingsAcceptingItems')
         # ask emergency for sending to cfg3
         item.setOtherMeetingConfigsClonableToEmergency((cfg3Id, ))
         self.assertEqual(
             item.displayOtherMeetingConfigsClonableTo(),
             unicode("{0} ({1}), {2} (<span class='item_clone_to_emergency'>Emergency</span> - {3})".format(
```

Ruff (0.15.11)

[error] Undefined name `unicode`

(F821)